cairo/tests/test_serde.py

```diff
@@ -263,11 +263,13 @@ def test_type(
             ExtendMemory,
             MessageCallOutput,
             Union[Bytes, LegacyTransaction],
+            Union[Bytes, Receipt],
```

Comment:
Add also Optional[Union[Bytes, Receipt]] and Optional[Union[Bytes, LegacyTransaction]] ?